kdumpctl

```diff
@@ -14,6 +14,7 @@ KDUMP_INITRD=""
 TARGET_INITRD=""
 #kdump shall be the default dump mode
 DEFAULT_DUMP_MODE="kdump"
+VMCORE_CREATION_STATUS="/var/crash/vmcore-creation.status"
```

Comment:
> Hi Philipp,
> 
> It doesn't matter what path the user set in kdump.conf, we just save the status file into /var/crash and done, as long as we can r/w it. I have no objection for "/var/lib/kdump", but I'm a little worry if selinux will prevent such writing, since for /var/crash, we already have the write permission, but as for "/var/lib/kdump", maybe we need to update the selinux rules as well. I haven't tested it though...

Hi @liutgnu ,  maybe no need to worry about  as selinux has fixed a issue here:
https://bugzilla.redhat.com/show_bug.cgi?id=1965985
https://github.com/fedora-selinux/selinux-policy/commit/af7e4b6492b315ee0912aef7f9f2d89a4f681ab6

> 
> Thanks, Tao Liu

Reply:
Hi @daveyoung & @prudo1 ,

Thanks for the info, please see v2 which I changed "/var/crash/vmcore-creation.status" to "/var/lib/kdump/vmcore-creation.status".